src/ray/rpc/node_manager/node_manager_server.h

```diff
@@ -70,6 +92,10 @@ class NodeManagerGrpcService : public GrpcService {
             main_service_));
 
     // Set accept concurrency.
+    server_call_factories_and_concurrencies->emplace_back(
+        std::move(submit_task_call_factory), 100);
+    server_call_factories_and_concurrencies->emplace_back(
+        std::move(wait_for_direct_actor_call_args_factory), 100);
```

Comment:
The concurrency should be much higher for this if we are going with this strategy.